tls/s2n_cipher_suites.c

```diff
@@ -219,6 +219,29 @@ struct s2n_cipher_suite s2n_null_cipher_suite = {
     .record_alg = &s2n_record_alg_null,
 };
 
+/* Both the group start and group end cipher suites are never negotiated. To ensure
+ * this, the IANA value is set to TLS_NULL_WITH_NULL_NULL. s2n checks that the negotiated
+ * cipher IANA id is never this value (see s2n_connection_get_cipher_iana_value) therefore preventing
+ * these ciphers from being negotiated.
+ */
+struct s2n_cipher_suite s2n_equal_preference_group_start = {
+    .available = 0,
+    .name = "EQUAL_PREFERENCE_GROUP_START",
+    .iana_value = { TLS_NULL_WITH_NULL_NULL },
+    .key_exchange_alg = &s2n_rsa,
+    .auth_method = S2N_AUTHENTICATION_RSA,
```

Comment:
I see 
```
.key_exchange_alg = NULL,
.auth_method = S2N_AUTHENTICATION_METHOD_TLS13,
```
I can update these fields but they are never used anyways? Is the request mainly consistency for the sake of consistency?

Reply:
Correct. Setting those fields to RSA might be misleading exactly because they're never used. None of the TLS1.3 cipher suites use those fields either, but setting them all to RSA would definitely be odd.